src/Resources/views/Admin/app.html.twig

```diff
@@ -83,14 +83,16 @@
 {% endverbatim %}
 </script>
 
-<div class="uie-panels js-uie-panels">
-    <div class="uie-panels__content js-uie-panels-selector">
+<script id="monsieurbiz-rich-editor-panels" type="x-tmpl-mustache">
+{% verbatim %}
+<div class="uie-panels js-uie-panels-{{uid}}">
+    <div class="uie-panels__content js-uie-panels-selector-{{uid}}">
         <div class="uie-flex uie-flex-main-between uie-w-full uie-mb-md">
-            <h2 class="ui header uie-heading uie-m-0" id="uie-heading">
+            <h2 class="ui header uie-heading uie-m-0" id="uie-heading-{{uid}}">
                 {{ 'monsieurbiz_richeditor_plugin.form.select_ui_element'|trans }}
```

Comment:
**❔ question:** Even if this section is `verbatim` the translation is made correctly?‏

Reply:
Good catch, I don't see any wording in this `h2` in our test env. We're checking right now.

Reply:
@jacquesbh The translation is not changed so it works.